.github/workflows/tests.yml

```diff
@@ -71,17 +71,23 @@ jobs:
       - name: Install nextest
         uses: taiki-e/install-action@nextest
       - name: Test forest
-        run: make -C forest test
+        run: |
+          cd forest
+          make test
```

Comment:
Why?

Reply:
Some of the scripts only works when CWD is forest root

Reply:
Tracking issue: https://github.com/ChainSafe/fil-actor-states/issues/165